server_addon/maya/server/settings/publishers.py

```diff
@@ -1371,6 +1600,52 @@ class PublishersModel(BaseSettingsModel):
     "ExtractModel": {
         "enabled": True,
         "optional": True,
-        "active": True,
+        "active": True
+    },
+    "ExtractAlembic": {
+        "enabled": True,
+        "families": [
+            "pointcache",
+            "model",
+            "vrayproxy.alembic"
+        ],
+        "attr": "",
+        "attrPrefix": "",
+        "autoSubd": False,
+        "bake_attributes": [],
+        "bake_attribute_prefixes": [],
+        "dataFormat": "ogawa",
+        "eulerFilter": False,
+        "melPerFrameCallback": "",
+        "melPostJobCallback": "",
+        "overrides": [
+            "attr",
+            "attrPrefix",
+            "renderableOnly",
+            "visibleOnly",
+            "worldSpace",
+            "writeColorSets",
+            "writeFaceSets",
+            "writeNormals"
+        ],
+        "preRoll": False,
+        "preRollStartFrame": 0,
+        "pythonPerFrameCallback": "",
+        "pythonPostJobCallback": "",
+        "renderableOnly": False,
+        "stripNamespaces": True,
+        "uvsOnly": False,
+        "uvWrite": False,
+        "userAttr": "",
+        "userAttrPrefix": "",
+        "verbose": False,
+        "visibleOnly": False,
+        "wholeFrameGeo": False,
+        "worldSpace": True,
+        "writeColorSets": False,
+        "writeFaceSets": False,
+        "writeNormals": True,
+        "writeUVSets": False,
+        "writeVisibility": False
```

Comment:
This defaulted to True before this PR - with this now False any animated visibilities (or even any visilities whatsoever) will not be published along. Do we feel that's a sensible default value for AYON?